mephisto/providers/mturk/mturk_worker.py

```diff
@@ -51,6 +51,20 @@ def __init__(
         )
         self._worker_name = self.worker_name  # sandbox workers use a different name
 
+    @classmethod
+    def get_from_mturk_worker_id(
+        cls, 
+        db: "MephistoDB", 
+        mturk_worker_id: str,
+    ) -> Optional["MTurkWorker"]:
+        """Get the MTurkWorker from the given worker_id"""
+        if cls.PROVIDER_TYPE != PROVIDER_TYPE:
+            mturk_worker_id += "_sandbox"
+        workers = db.find_workers(worker_name=mturk_worker_id, provider_type=cls.PROVIDER_TYPE)
+        if len(workers) == 0:
+            return None
```

Comment:
Hmm maybe print a warning here for clarity? Up to you

Reply:
Worth having as a `logger.info` I suppose, will mark it to add during the logging overhaul this half.